src/Cache/Cleaner.php

```diff
@@ -2,8 +2,47 @@
 
 namespace Timber\Cache;
 
+use Timber\Loader;
+
 class Cleaner
 {
+    public static function clear_cache($mode = 'all')
+    {
+        if (is_array($mode)) {
```

Comment:
Any reason `$mode` could be an array?

Reply:
Probably not. I updated the function signature in 062540dd359e8598aeb3cf3fd8a54d38bf2e21dd and added a note to the Upgrade Guide.